<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>6 Revisión | Intro RNA-seq LCG-UNAM 2022</title>
  <meta name="description" content="6 Revisión | Intro RNA-seq LCG-UNAM 2022" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="6 Revisión | Intro RNA-seq LCG-UNAM 2022" />
  <meta property="og:type" content="book" />
  
  
  
  <meta name="github-repo" content="lcolladotor/rnaseq_LCG-UNAM_2022" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="6 Revisión | Intro RNA-seq LCG-UNAM 2022" />
  
  
  

<meta name="author" content="Leonardo Collado-Torres" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  <link rel="shortcut icon" href="icon_32.png" type="image/x-icon" />
<link rel="prev" href="modelos-estadísticos.html"/>
<link rel="next" href="spatiallibd.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>


<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Overview</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#code-of-conduct"><i class="fa fa-check"></i>Code of Conduct</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#course-schedule"><i class="fa fa-check"></i>Course Schedule</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#external-links"><i class="fa fa-check"></i>External links</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#course-prerequisites"><i class="fa fa-check"></i>Course Prerequisites</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#course-evaluation"><i class="fa fa-check"></i>Course Evaluation</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#r-session-information"><i class="fa fa-check"></i>R session information</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#license"><i class="fa fa-check"></i>License</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="introducción-a-r-y-rstudio.html"><a href="introducción-a-r-y-rstudio.html"><i class="fa fa-check"></i><b>1</b> Introducción a R y RStudio</a>
<ul>
<li class="chapter" data-level="1.1" data-path="introducción-a-r-y-rstudio.html"><a href="introducción-a-r-y-rstudio.html#r"><i class="fa fa-check"></i><b>1.1</b> R</a></li>
<li class="chapter" data-level="1.2" data-path="introducción-a-r-y-rstudio.html"><a href="introducción-a-r-y-rstudio.html#github"><i class="fa fa-check"></i><b>1.2</b> GitHub</a></li>
<li class="chapter" data-level="1.3" data-path="introducción-a-r-y-rstudio.html"><a href="introducción-a-r-y-rstudio.html#rstudio"><i class="fa fa-check"></i><b>1.3</b> RStudio</a></li>
<li class="chapter" data-level="1.4" data-path="introducción-a-r-y-rstudio.html"><a href="introducción-a-r-y-rstudio.html#material-del-curso"><i class="fa fa-check"></i><b>1.4</b> Material del curso</a></li>
<li class="chapter" data-level="1.5" data-path="introducción-a-r-y-rstudio.html"><a href="introducción-a-r-y-rstudio.html#actividad-postcards-opcional"><i class="fa fa-check"></i><b>1.5</b> Actividad postcards (opcional)</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="introducción-a-bioconductor.html"><a href="introducción-a-bioconductor.html"><i class="fa fa-check"></i><b>2</b> Introducción a Bioconductor</a>
<ul>
<li class="chapter" data-level="2.1" data-path="introducción-a-bioconductor.html"><a href="introducción-a-bioconductor.html#encontrando-paquetes-de-bioconductor"><i class="fa fa-check"></i><b>2.1</b> Encontrando paquetes de Bioconductor</a></li>
<li class="chapter" data-level="2.2" data-path="introducción-a-bioconductor.html"><a href="introducción-a-bioconductor.html#estructura-de-un-paquete-de-bioc"><i class="fa fa-check"></i><b>2.2</b> Estructura de un paquete de BioC</a></li>
<li class="chapter" data-level="2.3" data-path="introducción-a-bioconductor.html"><a href="introducción-a-bioconductor.html#las-dos-ramas-de-bioconductor-release-y-devel"><i class="fa fa-check"></i><b>2.3</b> Las dos ramas de Bioconductor: release y devel</a></li>
<li class="chapter" data-level="2.4" data-path="introducción-a-bioconductor.html"><a href="introducción-a-bioconductor.html#cursos-y-eventos"><i class="fa fa-check"></i><b>2.4</b> Cursos y eventos</a></li>
<li class="chapter" data-level="2.5" data-path="introducción-a-bioconductor.html"><a href="introducción-a-bioconductor.html#comunidad"><i class="fa fa-check"></i><b>2.5</b> Comunidad</a></li>
<li class="chapter" data-level="2.6" data-path="introducción-a-bioconductor.html"><a href="introducción-a-bioconductor.html#ejercicio-grupal"><i class="fa fa-check"></i><b>2.6</b> Ejercicio grupal</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="objetos-de-bioconductor-para-datos-de-expresión.html"><a href="objetos-de-bioconductor-para-datos-de-expresión.html"><i class="fa fa-check"></i><b>3</b> Objetos de Bioconductor para datos de expresión</a>
<ul>
<li class="chapter" data-level="3.1" data-path="objetos-de-bioconductor-para-datos-de-expresión.html"><a href="objetos-de-bioconductor-para-datos-de-expresión.html#summarizedexperiment"><i class="fa fa-check"></i><b>3.1</b> SummarizedExperiment</a></li>
<li class="chapter" data-level="3.2" data-path="objetos-de-bioconductor-para-datos-de-expresión.html"><a href="objetos-de-bioconductor-para-datos-de-expresión.html#ejercicio"><i class="fa fa-check"></i><b>3.2</b> Ejercicio</a></li>
<li class="chapter" data-level="3.3" data-path="objetos-de-bioconductor-para-datos-de-expresión.html"><a href="objetos-de-bioconductor-para-datos-de-expresión.html#isee"><i class="fa fa-check"></i><b>3.3</b> iSEE</a></li>
<li class="chapter" data-level="3.4" data-path="objetos-de-bioconductor-para-datos-de-expresión.html"><a href="objetos-de-bioconductor-para-datos-de-expresión.html#ejercicio-con-spatiallibd"><i class="fa fa-check"></i><b>3.4</b> Ejercicio con spatialLIBD</a></li>
<li class="chapter" data-level="3.5" data-path="objetos-de-bioconductor-para-datos-de-expresión.html"><a href="objetos-de-bioconductor-para-datos-de-expresión.html#comunidad-1"><i class="fa fa-check"></i><b>3.5</b> Comunidad</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="datos-de-rna-seq-a-través-de-recount3.html"><a href="datos-de-rna-seq-a-través-de-recount3.html"><i class="fa fa-check"></i><b>4</b> Datos de RNA-seq a través de recount3</a>
<ul>
<li class="chapter" data-level="4.1" data-path="datos-de-rna-seq-a-través-de-recount3.html"><a href="datos-de-rna-seq-a-través-de-recount3.html#procesar-datos-crudos-fastq"><i class="fa fa-check"></i><b>4.1</b> Procesar datos crudos (FASTQ)</a></li>
<li class="chapter" data-level="4.2" data-path="datos-de-rna-seq-a-través-de-recount3.html"><a href="datos-de-rna-seq-a-través-de-recount3.html#proyectos-recount"><i class="fa fa-check"></i><b>4.2</b> Proyectos recount</a></li>
<li class="chapter" data-level="4.3" data-path="datos-de-rna-seq-a-través-de-recount3.html"><a href="datos-de-rna-seq-a-través-de-recount3.html#usar-recount3"><i class="fa fa-check"></i><b>4.3</b> Usar recount3</a></li>
<li class="chapter" data-level="4.4" data-path="datos-de-rna-seq-a-través-de-recount3.html"><a href="datos-de-rna-seq-a-través-de-recount3.html#ejercicio-1"><i class="fa fa-check"></i><b>4.4</b> Ejercicio</a></li>
<li class="chapter" data-level="4.5" data-path="datos-de-rna-seq-a-través-de-recount3.html"><a href="datos-de-rna-seq-a-través-de-recount3.html#comunidad-2"><i class="fa fa-check"></i><b>4.5</b> Comunidad</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="modelos-estadísticos.html"><a href="modelos-estadísticos.html"><i class="fa fa-check"></i><b>5</b> Modelos estadísticos</a>
<ul>
<li class="chapter" data-level="5.1" data-path="modelos-estadísticos.html"><a href="modelos-estadísticos.html#exploremodelmatrix"><i class="fa fa-check"></i><b>5.1</b> ExploreModelMatrix</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="modelos-estadísticos.html"><a href="modelos-estadísticos.html#ejemplo-1"><i class="fa fa-check"></i><b>5.1.1</b> Ejemplo 1</a></li>
<li class="chapter" data-level="5.1.2" data-path="modelos-estadísticos.html"><a href="modelos-estadísticos.html#ejemplo-2"><i class="fa fa-check"></i><b>5.1.2</b> Ejemplo 2</a></li>
<li class="chapter" data-level="5.1.3" data-path="modelos-estadísticos.html"><a href="modelos-estadísticos.html#ejemplo-3"><i class="fa fa-check"></i><b>5.1.3</b> Ejemplo 3</a></li>
<li class="chapter" data-level="5.1.4" data-path="modelos-estadísticos.html"><a href="modelos-estadísticos.html#ejercicio-2"><i class="fa fa-check"></i><b>5.1.4</b> Ejercicio</a></li>
<li class="chapter" data-level="5.1.5" data-path="modelos-estadísticos.html"><a href="modelos-estadísticos.html#para-aprender-más"><i class="fa fa-check"></i><b>5.1.5</b> Para aprender más</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="modelos-estadísticos.html"><a href="modelos-estadísticos.html#datos-de-srp045638"><i class="fa fa-check"></i><b>5.2</b> Datos de SRP045638</a></li>
<li class="chapter" data-level="5.3" data-path="modelos-estadísticos.html"><a href="modelos-estadísticos.html#normalización-de-datos"><i class="fa fa-check"></i><b>5.3</b> Normalización de datos</a></li>
<li class="chapter" data-level="5.4" data-path="modelos-estadísticos.html"><a href="modelos-estadísticos.html#expresión-diferencial"><i class="fa fa-check"></i><b>5.4</b> Expresión diferencial</a></li>
<li class="chapter" data-level="5.5" data-path="modelos-estadísticos.html"><a href="modelos-estadísticos.html#visualizando-genes-de"><i class="fa fa-check"></i><b>5.5</b> Visualizando genes DE</a></li>
<li class="chapter" data-level="5.6" data-path="modelos-estadísticos.html"><a href="modelos-estadísticos.html#ejercicio-3"><i class="fa fa-check"></i><b>5.6</b> Ejercicio</a></li>
<li class="chapter" data-level="5.7" data-path="modelos-estadísticos.html"><a href="modelos-estadísticos.html#comunidad-3"><i class="fa fa-check"></i><b>5.7</b> Comunidad</a></li>
<li class="chapter" data-level="5.8" data-path="modelos-estadísticos.html"><a href="modelos-estadísticos.html#ejercicio-respuesta"><i class="fa fa-check"></i><b>5.8</b> Ejercicio: respuesta</a></li>
<li class="chapter" data-level="5.9" data-path="modelos-estadísticos.html"><a href="modelos-estadísticos.html#específicaciones-del-proyecto"><i class="fa fa-check"></i><b>5.9</b> Específicaciones del proyecto</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="revisión.html"><a href="revisión.html"><i class="fa fa-check"></i><b>6</b> Revisión</a>
<ul>
<li class="chapter" data-level="6.1" data-path="revisión.html"><a href="revisión.html#ejercicio-en-equipo"><i class="fa fa-check"></i><b>6.1</b> Ejercicio en equipo</a></li>
<li class="chapter" data-level="6.2" data-path="revisión.html"><a href="revisión.html#respuestas"><i class="fa fa-check"></i><b>6.2</b> Respuestas</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="spatiallibd.html"><a href="spatiallibd.html"><i class="fa fa-check"></i><b>7</b> spatialLIBD</a></li>
<li class="chapter" data-level="8" data-path="comunidad-4.html"><a href="comunidad-4.html"><i class="fa fa-check"></i><b>8</b> Comunidad</a>
<ul>
<li class="chapter" data-level="8.1" data-path="comunidad-4.html"><a href="comunidad-4.html#de-ustedes"><i class="fa fa-check"></i><b>8.1</b> De ustedes</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Intro RNA-seq LCG-UNAM 2022</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="revisión" class="section level1" number="6">
<h1><span class="header-section-number">6</span> Revisión</h1>
<ul>
<li>¿Debemos explorar las relaciones entre nuestras variables con información de nuestras muestras previo a hacer un análisis de expresión diferencial?</li>
<li>¿Por qué usamos el paquete <code>edgeR</code>?</li>
<li>¿Por qué es importante el argumento <code>sort.by</code> en <code>topTable()</code>?</li>
<li>¿Por qué es importante el argumento <code>coef</code> en <code>topTable()</code>?</li>
</ul>
<p>Usemos los datos de <a href="http://research.libd.org/SPEAQeasy-example/bootcamp_intro" class="uri">http://research.libd.org/SPEAQeasy-example/bootcamp_intro</a></p>
<div class="sourceCode" id="cb170"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb170-1"><a href="revisión.html#cb170-1" aria-hidden="true" tabindex="-1"></a>speaqeasy_data <span class="ot">&lt;-</span> <span class="fu">file.path</span>(<span class="fu">tempdir</span>(), <span class="st">&quot;rse_speaqeasy.RData&quot;</span>)</span>
<span id="cb170-2"><a href="revisión.html#cb170-2" aria-hidden="true" tabindex="-1"></a><span class="fu">download.file</span>(<span class="st">&quot;https://github.com/LieberInstitute/SPEAQeasy-example/blob/master/rse_speaqeasy.RData?raw=true&quot;</span>, speaqeasy_data, <span class="at">mode =</span> <span class="st">&quot;wb&quot;</span>)</span>
<span id="cb170-3"><a href="revisión.html#cb170-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;SummarizedExperiment&quot;</span>)</span>
<span id="cb170-4"><a href="revisión.html#cb170-4" aria-hidden="true" tabindex="-1"></a><span class="fu">load</span>(speaqeasy_data, <span class="at">verbose =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
<pre><code>## Loading objects:
##   rse_gene</code></pre>
<div class="sourceCode" id="cb172"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb172-1"><a href="revisión.html#cb172-1" aria-hidden="true" tabindex="-1"></a>rse_gene</span></code></pre></div>
<pre><code>## class: RangedSummarizedExperiment 
## dim: 60609 40 
## metadata(0):
## assays(1): counts
## rownames(60609): ENSG00000223972.5 ENSG00000227232.5 ... ENSG00000210195.2 ENSG00000210196.2
## rowData names(10): Length gencodeID ... NumTx gencodeTx
## colnames(40): R13896_H7JKMBBXX R13903_HCTYLBBXX ... R15120_HFY2MBBXX R15134_HFFGHBBXX
## colData names(67): SAMPLE_ID FQCbasicStats ... AgeDeath BrNum</code></pre>
<ul>
<li>¿Cuantos genes y muestras tenemos en estos datos?</li>
</ul>
<div id="ejercicio-en-equipo" class="section level2" number="6.1">
<h2><span class="header-section-number">6.1</span> Ejercicio en equipo</h2>
<ul>
<li>¿Hay diferencias en <code>totalAssignedGene</code> o <code>mitoRate</code> entre los grupos de diagnosis (<code>PrimaryDx</code>)?</li>
<li>Grafica la expresión de <em>SNAP25</em> para cada grupo de diagnosis.</li>
<li>Sugiere un modelo estadistico que podríamos usar en una análisis de expresión diferencial. Verifica que si sea un modelo <em>full rank</em>. ¿Cúal sería el o los coeficientes de interés?</li>
</ul>
</div>
<div id="respuestas" class="section level2" number="6.2">
<h2><span class="header-section-number">6.2</span> Respuestas</h2>
<div class="sourceCode" id="cb174"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb174-1"><a href="revisión.html#cb174-1" aria-hidden="true" tabindex="-1"></a><span class="do">## Exploremos la variable de PrimaryDx</span></span>
<span id="cb174-2"><a href="revisión.html#cb174-2" aria-hidden="true" tabindex="-1"></a><span class="fu">table</span>(rse_gene<span class="sc">$</span>PrimaryDx)</span></code></pre></div>
<pre><code>## 
## Bipolar Control   Other 
##      20      20       0</code></pre>
<div class="sourceCode" id="cb176"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb176-1"><a href="revisión.html#cb176-1" aria-hidden="true" tabindex="-1"></a><span class="do">## Eliminemos el diagnosis &quot;Other&quot; porque no tiene información</span></span>
<span id="cb176-2"><a href="revisión.html#cb176-2" aria-hidden="true" tabindex="-1"></a>rse_gene<span class="sc">$</span>PrimaryDx <span class="ot">&lt;-</span> <span class="fu">droplevels</span>(rse_gene<span class="sc">$</span>PrimaryDx)</span>
<span id="cb176-3"><a href="revisión.html#cb176-3" aria-hidden="true" tabindex="-1"></a><span class="fu">table</span>(rse_gene<span class="sc">$</span>PrimaryDx)</span></code></pre></div>
<pre><code>## 
## Bipolar Control 
##      20      20</code></pre>
<div class="sourceCode" id="cb178"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb178-1"><a href="revisión.html#cb178-1" aria-hidden="true" tabindex="-1"></a><span class="do">## Exploremos numéricamente diferencias entre grupos de diagnosis para</span></span>
<span id="cb178-2"><a href="revisión.html#cb178-2" aria-hidden="true" tabindex="-1"></a><span class="do">## varias variables</span></span>
<span id="cb178-3"><a href="revisión.html#cb178-3" aria-hidden="true" tabindex="-1"></a><span class="fu">with</span>(<span class="fu">colData</span>(rse_gene), <span class="fu">tapply</span>(totalAssignedGene, PrimaryDx, summary))</span></code></pre></div>
<pre><code>## $Bipolar
##    Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
## 0.09529 0.24454 0.29773 0.28867 0.34662 0.38358 
## 
## $Control
##    Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
##  0.1384  0.2575  0.2895  0.2984  0.3537  0.4073</code></pre>
<div class="sourceCode" id="cb180"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb180-1"><a href="revisión.html#cb180-1" aria-hidden="true" tabindex="-1"></a><span class="fu">with</span>(<span class="fu">colData</span>(rse_gene), <span class="fu">tapply</span>(mitoRate, PrimaryDx, summary))</span></code></pre></div>
<pre><code>## $Bipolar
##    Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
## 0.01297 0.01555 0.01735 0.01818 0.01980 0.03019 
## 
## $Control
##    Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
## 0.01194 0.01564 0.01911 0.02029 0.02439 0.03037</code></pre>
<div class="sourceCode" id="cb182"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb182-1"><a href="revisión.html#cb182-1" aria-hidden="true" tabindex="-1"></a><span class="do">## Podemos hacer lo mismo para otras variables</span></span>
<span id="cb182-2"><a href="revisión.html#cb182-2" aria-hidden="true" tabindex="-1"></a><span class="fu">with</span>(<span class="fu">colData</span>(rse_gene), <span class="fu">tapply</span>(mitoRate, BrainRegion, summary))</span></code></pre></div>
<pre><code>## $Amygdala
##    Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
## 0.01302 0.01706 0.02029 0.02097 0.02437 0.03037 
## 
## $sACC
##    Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
## 0.01194 0.01449 0.01686 0.01749 0.01836 0.03019</code></pre>
<div class="sourceCode" id="cb184"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb184-1"><a href="revisión.html#cb184-1" aria-hidden="true" tabindex="-1"></a><span class="do">## Podemos resolver la primeras preguntas con iSEE</span></span>
<span id="cb184-2"><a href="revisión.html#cb184-2" aria-hidden="true" tabindex="-1"></a><span class="cf">if</span> (<span class="fu">interactive</span>()) iSEE<span class="sc">::</span><span class="fu">iSEE</span>(rse_gene)</span>
<span id="cb184-3"><a href="revisión.html#cb184-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb184-4"><a href="revisión.html#cb184-4" aria-hidden="true" tabindex="-1"></a><span class="do">## O hacer graficas nosotros mismos. Aquí les muestro una posible respuesta</span></span>
<span id="cb184-5"><a href="revisión.html#cb184-5" aria-hidden="true" tabindex="-1"></a><span class="do">## con ggplot2</span></span>
<span id="cb184-6"><a href="revisión.html#cb184-6" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;ggplot2&quot;</span>)</span>
<span id="cb184-7"><a href="revisión.html#cb184-7" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(</span>
<span id="cb184-8"><a href="revisión.html#cb184-8" aria-hidden="true" tabindex="-1"></a>    <span class="fu">as.data.frame</span>(<span class="fu">colData</span>(rse_gene)),</span>
<span id="cb184-9"><a href="revisión.html#cb184-9" aria-hidden="true" tabindex="-1"></a>    <span class="fu">aes</span>(<span class="at">y =</span> totalAssignedGene, <span class="at">group =</span> PrimaryDx, <span class="at">x =</span> PrimaryDx)</span>
<span id="cb184-10"><a href="revisión.html#cb184-10" aria-hidden="true" tabindex="-1"></a>) <span class="sc">+</span></span>
<span id="cb184-11"><a href="revisión.html#cb184-11" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_boxplot</span>() <span class="sc">+</span></span>
<span id="cb184-12"><a href="revisión.html#cb184-12" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme_bw</span>(<span class="at">base_size =</span> <span class="dv">20</span>) <span class="sc">+</span></span>
<span id="cb184-13"><a href="revisión.html#cb184-13" aria-hidden="true" tabindex="-1"></a>    <span class="fu">xlab</span>(<span class="st">&quot;Diagnosis&quot;</span>)</span></code></pre></div>
<p><img src="_main_files/figure-html/respuestas-1.png" width="672" /></p>
<div class="sourceCode" id="cb185"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb185-1"><a href="revisión.html#cb185-1" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(</span>
<span id="cb185-2"><a href="revisión.html#cb185-2" aria-hidden="true" tabindex="-1"></a>    <span class="fu">as.data.frame</span>(<span class="fu">colData</span>(rse_gene)),</span>
<span id="cb185-3"><a href="revisión.html#cb185-3" aria-hidden="true" tabindex="-1"></a>    <span class="fu">aes</span>(<span class="at">y =</span> mitoRate, <span class="at">group =</span> PrimaryDx, <span class="at">x =</span> PrimaryDx)</span>
<span id="cb185-4"><a href="revisión.html#cb185-4" aria-hidden="true" tabindex="-1"></a>) <span class="sc">+</span></span>
<span id="cb185-5"><a href="revisión.html#cb185-5" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_boxplot</span>() <span class="sc">+</span></span>
<span id="cb185-6"><a href="revisión.html#cb185-6" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme_bw</span>(<span class="at">base_size =</span> <span class="dv">20</span>) <span class="sc">+</span></span>
<span id="cb185-7"><a href="revisión.html#cb185-7" aria-hidden="true" tabindex="-1"></a>    <span class="fu">xlab</span>(<span class="st">&quot;Diagnosis&quot;</span>)</span></code></pre></div>
<p><img src="_main_files/figure-html/respuestas-2.png" width="672" /></p>
<div class="sourceCode" id="cb186"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb186-1"><a href="revisión.html#cb186-1" aria-hidden="true" tabindex="-1"></a><span class="do">## Otras variables</span></span>
<span id="cb186-2"><a href="revisión.html#cb186-2" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(</span>
<span id="cb186-3"><a href="revisión.html#cb186-3" aria-hidden="true" tabindex="-1"></a>    <span class="fu">as.data.frame</span>(<span class="fu">colData</span>(rse_gene)),</span>
<span id="cb186-4"><a href="revisión.html#cb186-4" aria-hidden="true" tabindex="-1"></a>    <span class="fu">aes</span>(<span class="at">y =</span> mitoRate, <span class="at">group =</span> BrainRegion, <span class="at">x =</span> BrainRegion)</span>
<span id="cb186-5"><a href="revisión.html#cb186-5" aria-hidden="true" tabindex="-1"></a>) <span class="sc">+</span></span>
<span id="cb186-6"><a href="revisión.html#cb186-6" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_boxplot</span>() <span class="sc">+</span></span>
<span id="cb186-7"><a href="revisión.html#cb186-7" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme_bw</span>(<span class="at">base_size =</span> <span class="dv">20</span>) <span class="sc">+</span></span>
<span id="cb186-8"><a href="revisión.html#cb186-8" aria-hidden="true" tabindex="-1"></a>    <span class="fu">xlab</span>(<span class="st">&quot;Brain Region&quot;</span>)</span></code></pre></div>
<p><img src="_main_files/figure-html/respuestas-3.png" width="672" /></p>
<div class="sourceCode" id="cb187"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb187-1"><a href="revisión.html#cb187-1" aria-hidden="true" tabindex="-1"></a><span class="do">## Encontremos el gene SNAP25</span></span>
<span id="cb187-2"><a href="revisión.html#cb187-2" aria-hidden="true" tabindex="-1"></a><span class="fu">rowRanges</span>(rse_gene)</span></code></pre></div>
<pre><code>## GRanges object with 60609 ranges and 10 metadata columns:
##                     seqnames      ranges strand |    Length         gencodeID       ensemblID              gene_type
##                        &lt;Rle&gt;   &lt;IRanges&gt;  &lt;Rle&gt; | &lt;integer&gt;       &lt;character&gt;     &lt;character&gt;            &lt;character&gt;
##   ENSG00000223972.5     chr1 11869-13670      + |      1735 ENSG00000223972.5 ENSG00000223972 transcribed_unproces..
##   ENSG00000227232.5     chr1 14404-29570      - |      1351 ENSG00000227232.5 ENSG00000227232 unprocessed_pseudogene
##   ENSG00000278267.1     chr1 17369-17436      - |        68 ENSG00000278267.1 ENSG00000278267                  miRNA
##   ENSG00000243485.5     chr1 29554-31097      + |      1021 ENSG00000243485.5 ENSG00000243485                 lncRNA
##   ENSG00000284332.1     chr1 30366-30503      + |       138 ENSG00000284332.1 ENSG00000284332                  miRNA
##                 ...      ...         ...    ... .       ...               ...             ...                    ...
##   ENSG00000198695.2     chrM 14149-14673      - |       525 ENSG00000198695.2 ENSG00000198695         protein_coding
##   ENSG00000210194.1     chrM 14674-14742      - |        69 ENSG00000210194.1 ENSG00000210194                Mt_tRNA
##   ENSG00000198727.2     chrM 14747-15887      + |      1141 ENSG00000198727.2 ENSG00000198727         protein_coding
##   ENSG00000210195.2     chrM 15888-15953      + |        66 ENSG00000210195.2 ENSG00000210195                Mt_tRNA
##   ENSG00000210196.2     chrM 15956-16023      - |        68 ENSG00000210196.2 ENSG00000210196                Mt_tRNA
##                          Symbol  EntrezID       Class  meanExprs     NumTx              gencodeTx
##                     &lt;character&gt; &lt;integer&gt; &lt;character&gt;  &lt;numeric&gt; &lt;integer&gt;            &lt;character&gt;
##   ENSG00000223972.5     DDX11L1      &lt;NA&gt;       InGen 0.00347263         2 ENST00000456328.2;EN..
##   ENSG00000227232.5      WASH7P      &lt;NA&gt;       InGen 1.57167877         1      ENST00000488147.1
##   ENSG00000278267.1   MIR6859-1      &lt;NA&gt;       InGen 7.74437941         1      ENST00000619216.1
##   ENSG00000243485.5 MIR1302-2HG      &lt;NA&gt;       InGen 0.01403484         2 ENST00000473358.1;EN..
##   ENSG00000284332.1   MIR1302-2      &lt;NA&gt;       InGen 0.00000000         1      ENST00000607096.1
##                 ...         ...       ...         ...        ...       ...                    ...
##   ENSG00000198695.2      MT-ND6      4541       InGen  182.92863         1      ENST00000361681.2
##   ENSG00000210194.1       MT-TE      &lt;NA&gt;       InGen    9.80684         1      ENST00000387459.1
##   ENSG00000198727.2      MT-CYB      4519       InGen  826.87424         1      ENST00000361789.2
##   ENSG00000210195.2       MT-TT      &lt;NA&gt;       InGen  276.37225         1      ENST00000387460.2
##   ENSG00000210196.2       MT-TP      &lt;NA&gt;       InGen  366.58138         1      ENST00000387461.2
##   -------
##   seqinfo: 25 sequences from an unspecified genome; no seqlengths</code></pre>
<div class="sourceCode" id="cb189"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb189-1"><a href="revisión.html#cb189-1" aria-hidden="true" tabindex="-1"></a><span class="do">## En este objeto los nombres de los genes vienen en la variable &quot;Symbol&quot;</span></span>
<span id="cb189-2"><a href="revisión.html#cb189-2" aria-hidden="true" tabindex="-1"></a>i <span class="ot">&lt;-</span> <span class="fu">which</span>(<span class="fu">rowRanges</span>(rse_gene)<span class="sc">$</span>Symbol <span class="sc">==</span> <span class="st">&quot;SNAP25&quot;</span>)</span>
<span id="cb189-3"><a href="revisión.html#cb189-3" aria-hidden="true" tabindex="-1"></a>i</span></code></pre></div>
<pre><code>## [1] 54098</code></pre>
<div class="sourceCode" id="cb191"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb191-1"><a href="revisión.html#cb191-1" aria-hidden="true" tabindex="-1"></a><span class="do">## Para graficar con ggplot2, hagamos un pequeño data.frame</span></span>
<span id="cb191-2"><a href="revisión.html#cb191-2" aria-hidden="true" tabindex="-1"></a>df <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb191-3"><a href="revisión.html#cb191-3" aria-hidden="true" tabindex="-1"></a>    <span class="at">expression =</span> <span class="fu">assay</span>(rse_gene)[i, ],</span>
<span id="cb191-4"><a href="revisión.html#cb191-4" aria-hidden="true" tabindex="-1"></a>    <span class="at">Dx =</span> rse_gene<span class="sc">$</span>PrimaryDx</span>
<span id="cb191-5"><a href="revisión.html#cb191-5" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb191-6"><a href="revisión.html#cb191-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb191-7"><a href="revisión.html#cb191-7" aria-hidden="true" tabindex="-1"></a><span class="do">## Ya teniendo el pequeño data.frame, podemos hacer la gráfica</span></span>
<span id="cb191-8"><a href="revisión.html#cb191-8" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(df, <span class="fu">aes</span>(<span class="at">y =</span> <span class="fu">log2</span>(expression <span class="sc">+</span> <span class="fl">0.5</span>), <span class="at">group =</span> Dx, <span class="at">x =</span> Dx)) <span class="sc">+</span></span>
<span id="cb191-9"><a href="revisión.html#cb191-9" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_boxplot</span>() <span class="sc">+</span></span>
<span id="cb191-10"><a href="revisión.html#cb191-10" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme_bw</span>(<span class="at">base_size =</span> <span class="dv">20</span>) <span class="sc">+</span></span>
<span id="cb191-11"><a href="revisión.html#cb191-11" aria-hidden="true" tabindex="-1"></a>    <span class="fu">xlab</span>(<span class="st">&quot;Diagnosis&quot;</span>) <span class="sc">+</span></span>
<span id="cb191-12"><a href="revisión.html#cb191-12" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ylab</span>(<span class="st">&quot;SNAP25: log2(x + 0.5)&quot;</span>)</span></code></pre></div>
<p><img src="_main_files/figure-html/respuestas-4.png" width="672" /></p>
<div class="sourceCode" id="cb192"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb192-1"><a href="revisión.html#cb192-1" aria-hidden="true" tabindex="-1"></a><span class="do">## https://bioconductor.org/packages/release/bioc/vignettes/scater/inst/doc/overview.html#3_Visualizing_expression_values</span></span>
<span id="cb192-2"><a href="revisión.html#cb192-2" aria-hidden="true" tabindex="-1"></a>scater<span class="sc">::</span><span class="fu">plotExpression</span>(</span>
<span id="cb192-3"><a href="revisión.html#cb192-3" aria-hidden="true" tabindex="-1"></a>    <span class="fu">as</span>(rse_gene, <span class="st">&quot;SingleCellExperiment&quot;</span>),</span>
<span id="cb192-4"><a href="revisión.html#cb192-4" aria-hidden="true" tabindex="-1"></a>    <span class="at">features =</span> <span class="fu">rownames</span>(rse_gene)[i],</span>
<span id="cb192-5"><a href="revisión.html#cb192-5" aria-hidden="true" tabindex="-1"></a>    <span class="at">x =</span> <span class="st">&quot;PrimaryDx&quot;</span>,</span>
<span id="cb192-6"><a href="revisión.html#cb192-6" aria-hidden="true" tabindex="-1"></a>    <span class="at">exprs_values =</span> <span class="st">&quot;counts&quot;</span>,</span>
<span id="cb192-7"><a href="revisión.html#cb192-7" aria-hidden="true" tabindex="-1"></a>    <span class="at">colour_by =</span> <span class="st">&quot;BrainRegion&quot;</span>,</span>
<span id="cb192-8"><a href="revisión.html#cb192-8" aria-hidden="true" tabindex="-1"></a>    <span class="at">xlab =</span> <span class="st">&quot;Diagnosis&quot;</span></span>
<span id="cb192-9"><a href="revisión.html#cb192-9" aria-hidden="true" tabindex="-1"></a>)</span></code></pre></div>
<p><img src="_main_files/figure-html/respuestas-5.png" width="672" /></p>
<div class="sourceCode" id="cb193"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb193-1"><a href="revisión.html#cb193-1" aria-hidden="true" tabindex="-1"></a><span class="do">## Para el model estadístico exploremos la información de las muestras</span></span>
<span id="cb193-2"><a href="revisión.html#cb193-2" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(<span class="fu">colData</span>(rse_gene))</span></code></pre></div>
<pre><code>##  [1] &quot;SAMPLE_ID&quot;                          &quot;FQCbasicStats&quot;                      &quot;perBaseQual&quot;                       
##  [4] &quot;perTileQual&quot;                        &quot;perSeqQual&quot;                         &quot;perBaseContent&quot;                    
##  [7] &quot;GCcontent&quot;                          &quot;Ncontent&quot;                           &quot;SeqLengthDist&quot;                     
## [10] &quot;SeqDuplication&quot;                     &quot;OverrepSeqs&quot;                        &quot;AdapterContent&quot;                    
## [13] &quot;KmerContent&quot;                        &quot;SeqLength_R1&quot;                       &quot;percentGC_R1&quot;                      
## [16] &quot;phred20-21_R1&quot;                      &quot;phred48-49_R1&quot;                      &quot;phred76-77_R1&quot;                     
## [19] &quot;phred100-101_R1&quot;                    &quot;phredGT30_R1&quot;                       &quot;phredGT35_R1&quot;                      
## [22] &quot;Adapter50-51_R1&quot;                    &quot;Adapter70-71_R1&quot;                    &quot;Adapter90_R1&quot;                      
## [25] &quot;SeqLength_R2&quot;                       &quot;percentGC_R2&quot;                       &quot;phred20-21_R2&quot;                     
## [28] &quot;phred48-49_R2&quot;                      &quot;phred76-77_R2&quot;                      &quot;phred100-101_R2&quot;                   
## [31] &quot;phredGT30_R2&quot;                       &quot;phredGT35_R2&quot;                       &quot;Adapter50-51_R2&quot;                   
## [34] &quot;Adapter70-71_R2&quot;                    &quot;Adapter90_R2&quot;                       &quot;bamFile&quot;                           
## [37] &quot;trimmed&quot;                            &quot;numReads&quot;                           &quot;numMapped&quot;                         
## [40] &quot;numUnmapped&quot;                        &quot;overallMapRate&quot;                     &quot;concordMapRate&quot;                    
## [43] &quot;totalMapped&quot;                        &quot;mitoMapped&quot;                         &quot;mitoRate&quot;                          
## [46] &quot;totalAssignedGene&quot;                  &quot;gene_Assigned&quot;                      &quot;gene_Unassigned_Unmapped&quot;          
## [49] &quot;gene_Unassigned_Read_Type&quot;          &quot;gene_Unassigned_Singleton&quot;          &quot;gene_Unassigned_MappingQuality&quot;    
## [52] &quot;gene_Unassigned_Chimera&quot;            &quot;gene_Unassigned_FragmentLength&quot;     &quot;gene_Unassigned_Duplicate&quot;         
## [55] &quot;gene_Unassigned_MultiMapping&quot;       &quot;gene_Unassigned_Secondary&quot;          &quot;gene_Unassigned_NonSplit&quot;          
## [58] &quot;gene_Unassigned_NoFeatures&quot;         &quot;gene_Unassigned_Overlapping_Length&quot; &quot;gene_Unassigned_Ambiguity&quot;         
## [61] &quot;rRNA_rate&quot;                          &quot;BrainRegion&quot;                        &quot;Race&quot;                              
## [64] &quot;PrimaryDx&quot;                          &quot;Sex&quot;                                &quot;AgeDeath&quot;                          
## [67] &quot;BrNum&quot;</code></pre>
<div class="sourceCode" id="cb195"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb195-1"><a href="revisión.html#cb195-1" aria-hidden="true" tabindex="-1"></a><span class="do">## Podemos usar región del cerebro porque tenemos suficientes datos</span></span>
<span id="cb195-2"><a href="revisión.html#cb195-2" aria-hidden="true" tabindex="-1"></a><span class="fu">table</span>(rse_gene<span class="sc">$</span>BrainRegion)</span></code></pre></div>
<pre><code>## 
## Amygdala     sACC 
##       20       20</code></pre>
<div class="sourceCode" id="cb197"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb197-1"><a href="revisión.html#cb197-1" aria-hidden="true" tabindex="-1"></a><span class="do">## Pero no podemos usar &quot;Race&quot; porque son solo de 1 tipo</span></span>
<span id="cb197-2"><a href="revisión.html#cb197-2" aria-hidden="true" tabindex="-1"></a><span class="fu">table</span>(rse_gene<span class="sc">$</span>Race)</span></code></pre></div>
<pre><code>## 
## CAUC 
##   40</code></pre>
<div class="sourceCode" id="cb199"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb199-1"><a href="revisión.html#cb199-1" aria-hidden="true" tabindex="-1"></a><span class="do">## Ojo! Acá es importante que hayamos usado droplevels(rse_gene$PrimaryDx)</span></span>
<span id="cb199-2"><a href="revisión.html#cb199-2" aria-hidden="true" tabindex="-1"></a><span class="do">## si no, vamos a tener un modelo que no sea _full rank_</span></span>
<span id="cb199-3"><a href="revisión.html#cb199-3" aria-hidden="true" tabindex="-1"></a>mod <span class="ot">&lt;-</span> <span class="fu">with</span>(</span>
<span id="cb199-4"><a href="revisión.html#cb199-4" aria-hidden="true" tabindex="-1"></a>    <span class="fu">colData</span>(rse_gene),</span>
<span id="cb199-5"><a href="revisión.html#cb199-5" aria-hidden="true" tabindex="-1"></a>    <span class="fu">model.matrix</span>(<span class="sc">~</span> PrimaryDx <span class="sc">+</span> totalAssignedGene <span class="sc">+</span> mitoRate <span class="sc">+</span> rRNA_rate <span class="sc">+</span> BrainRegion <span class="sc">+</span> Sex <span class="sc">+</span> AgeDeath)</span>
<span id="cb199-6"><a href="revisión.html#cb199-6" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb199-7"><a href="revisión.html#cb199-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb199-8"><a href="revisión.html#cb199-8" aria-hidden="true" tabindex="-1"></a><span class="do">## Exploremos el modelo de forma interactiva</span></span>
<span id="cb199-9"><a href="revisión.html#cb199-9" aria-hidden="true" tabindex="-1"></a><span class="cf">if</span> (<span class="fu">interactive</span>()) {</span>
<span id="cb199-10"><a href="revisión.html#cb199-10" aria-hidden="true" tabindex="-1"></a>    <span class="do">## Tenemos que eliminar columnas que tienen NAs.</span></span>
<span id="cb199-11"><a href="revisión.html#cb199-11" aria-hidden="true" tabindex="-1"></a>    info_no_NAs <span class="ot">&lt;-</span> <span class="fu">colData</span>(rse_gene)[, <span class="fu">c</span>(</span>
<span id="cb199-12"><a href="revisión.html#cb199-12" aria-hidden="true" tabindex="-1"></a>        <span class="st">&quot;PrimaryDx&quot;</span>, <span class="st">&quot;totalAssignedGene&quot;</span>, <span class="st">&quot;rRNA_rate&quot;</span>, <span class="st">&quot;BrainRegion&quot;</span>, <span class="st">&quot;Sex&quot;</span>,</span>
<span id="cb199-13"><a href="revisión.html#cb199-13" aria-hidden="true" tabindex="-1"></a>        <span class="st">&quot;AgeDeath&quot;</span>, <span class="st">&quot;mitoRate&quot;</span>, <span class="st">&quot;Race&quot;</span></span>
<span id="cb199-14"><a href="revisión.html#cb199-14" aria-hidden="true" tabindex="-1"></a>    )]</span>
<span id="cb199-15"><a href="revisión.html#cb199-15" aria-hidden="true" tabindex="-1"></a>    ExploreModelMatrix<span class="sc">::</span><span class="fu">ExploreModelMatrix</span>(</span>
<span id="cb199-16"><a href="revisión.html#cb199-16" aria-hidden="true" tabindex="-1"></a>        info_no_NAs,</span>
<span id="cb199-17"><a href="revisión.html#cb199-17" aria-hidden="true" tabindex="-1"></a>        <span class="sc">~</span> PrimaryDx <span class="sc">+</span> totalAssignedGene <span class="sc">+</span> mitoRate <span class="sc">+</span> rRNA_rate <span class="sc">+</span> BrainRegion <span class="sc">+</span> Sex <span class="sc">+</span> AgeDeath</span>
<span id="cb199-18"><a href="revisión.html#cb199-18" aria-hidden="true" tabindex="-1"></a>    )</span>
<span id="cb199-19"><a href="revisión.html#cb199-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb199-20"><a href="revisión.html#cb199-20" aria-hidden="true" tabindex="-1"></a>    <span class="do">## Veamos un modelo más sencillo sin las variables numéricas (continuas) porque</span></span>
<span id="cb199-21"><a href="revisión.html#cb199-21" aria-hidden="true" tabindex="-1"></a>    <span class="do">## ExploreModelMatrix nos las muestra como si fueran factors (categoricas)</span></span>
<span id="cb199-22"><a href="revisión.html#cb199-22" aria-hidden="true" tabindex="-1"></a>    <span class="do">## en vez de continuas</span></span>
<span id="cb199-23"><a href="revisión.html#cb199-23" aria-hidden="true" tabindex="-1"></a>    ExploreModelMatrix<span class="sc">::</span><span class="fu">ExploreModelMatrix</span>(</span>
<span id="cb199-24"><a href="revisión.html#cb199-24" aria-hidden="true" tabindex="-1"></a>        info_no_NAs,</span>
<span id="cb199-25"><a href="revisión.html#cb199-25" aria-hidden="true" tabindex="-1"></a>        <span class="sc">~</span> PrimaryDx <span class="sc">+</span> BrainRegion <span class="sc">+</span> Sex</span>
<span id="cb199-26"><a href="revisión.html#cb199-26" aria-hidden="true" tabindex="-1"></a>    )</span>
<span id="cb199-27"><a href="revisión.html#cb199-27" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb199-28"><a href="revisión.html#cb199-28" aria-hidden="true" tabindex="-1"></a>    <span class="do">## Si agregamos + Race nos da errores porque Race solo tiene 1 opción</span></span>
<span id="cb199-29"><a href="revisión.html#cb199-29" aria-hidden="true" tabindex="-1"></a>    <span class="co"># ExploreModelMatrix::ExploreModelMatrix(</span></span>
<span id="cb199-30"><a href="revisión.html#cb199-30" aria-hidden="true" tabindex="-1"></a>    <span class="co">#     info_no_NAs,</span></span>
<span id="cb199-31"><a href="revisión.html#cb199-31" aria-hidden="true" tabindex="-1"></a>    <span class="co">#     ~ PrimaryDx + BrainRegion + Sex + Race</span></span>
<span id="cb199-32"><a href="revisión.html#cb199-32" aria-hidden="true" tabindex="-1"></a>    <span class="co"># )</span></span>
<span id="cb199-33"><a href="revisión.html#cb199-33" aria-hidden="true" tabindex="-1"></a>}</span></code></pre></div>
<p>¿Quieres más datos? Tenemos muchos en LIBD incluyendo <a href="http://eqtl.brainseq.org/phase2/" class="uri">http://eqtl.brainseq.org/phase2/</a>.</p>
</div>
</div>
<div class="container">
    <footer class="site-footer">

      <center>
<p class="powered-by">
© 2011-2020. All thoughts and opinions here are my own. The icon was designed by <a href="https://www.linkedin.com/in/mauricio-guzman-6529b551/">Mauricio Guzmán</a> and is inspired by <a href="https://en.wikipedia.org/wiki/Huichol">Huichol culture</a>; it represents my community building interests.
</p>

<p class="powered-by copyright-license-text">
This work is licensed under <a href="https://creativecommons.org/licenses/by-nc-sa/4.0" rel="noopener noreferrer" target="_blank">CC BY NC SA 4.0</a>
</p>

<p class="powered-by footer-license-icons">
    <a href="https://creativecommons.org/licenses/by-nc-sa/4.0" rel="noopener noreferrer" target="_blank">
      <img src="https://search.creativecommons.org/static/img/cc_icon.svg" alt="CC icon" width="32px">
      <img src="https://search.creativecommons.org/static/img/cc-by_icon.svg" alt="CC by icon" width="32px">
      <img src="https://search.creativecommons.org/static/img/cc-nc_icon.svg" alt="CC NC icon" width="32px">
      <img src="https://search.creativecommons.org/static/img/cc-sa_icon.svg" alt="CC SA icon" width="32px">
    </a>
</p>

<div id="clustrmaps-widget"></div><script type='text/javascript' id='clustrmaps' src='//cdn.clustrmaps.com/map_v2.js?cl=ffffff&w=300&t=m&d=Zo-deincaJGsqhqjIWsujtdoNOMJsnnZLefKmxDKVv8&co=124b73&cmo=dd53ff&cmn=4ec724'></script>

<p class="powered-by">
Published with <a href="https://bookdown.org/" target="_blank" rel="noopener">Bookdown</a>
</p>

</center>



<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-45299226-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-45299226-1');
</script>

</footer>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="modelos-estadísticos.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="spatiallibd.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
